cmd/tailscale/cli/configure-kube.go

```diff
@@ -26,12 +26,16 @@ func init() {
 
 var configureKubeconfigCmd = &ffcli.Command{
 	Name:       "kubeconfig",
-	ShortHelp:  "Configure kubeconfig to use Tailscale",
+	ShortHelp:  "Configure kubectl to connect to a Tailscale Kubernetes Auth Proxy",
 	ShortUsage: "kubeconfig <hostname-or-fqdn>",
 	LongHelp: strings.TrimSpace(`
-Run this command to configure your kubeconfig to use Tailscale for authentication to a Kubernetes cluster.
+*** ALPHA; all of this is subject to change ***
+
+Run this command to configure kubectl to connect to a Kubernetes cluster over Tailscale.
 
 The hostname argument should be set to the Tailscale hostname of the peer running as an auth proxy in the cluster.
+
+See: https://tailscale.com/kb/1236/kubernetes-operator/
```

Comment:
Recommend adding a shortlink in tailscale-www/tsweb/redirects.txt so we don't hard-code KB 1236.

Reply:
ack

Reply:
will do before merging